dbt/models/mart/dim_listing.sql

```diff
@@ -0,0 +1,30 @@
+{{ config(materialized='table') }}
+
+WITH l1 AS (
+    select * from {{ ref('all_available_listings') }}
+),
+
+l2 AS (
```

Comment:
You shouldn't need to call out the same table twice, since the below is a self join and you can reference the same table multiple times.

No need to change this - but want to make sure you were doing this intentionally for structure/formatting instead of for SQL runtime needs :)